### PR TITLE
created profile

### PR DESCRIPTION
**Pull request by theheckle**

### CODE REVIEW

**Comment by ritjoe** [comment]
:+1:

File: people/2017/spring/sheckle.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#hfoss
+name: Sam Heckle
+avatar: https://secure.gravatar.com/avatar/911288192aeb67d906b3ce8867b98f8a
+blog: https://samheckle.wordpress.com/
+feed: https://samheckle.wordpress.com/feed/
+email: sxh4716@rit.edu
+major: Game Design and Development
+irc: sheckle
+rit_dce: None
```
Comment (jflory7) [inline people/2017/spring/sheckle.yaml]: @theheckle @ritjoe Also noticed it for this one too, `rit_dce` is blank in this profile too. For reference, this is your RIT computer account / username, e.g. sxh4716. You will probably want to send in another pull request to add this into your profile, @theheckle. 😃
